cloudprem/modules/replicated/variables.tf

```diff
@@ -5,4 +5,8 @@ variable "dozuki_license_parameter_name" {
 variable "nlb_hostname" {
   description = "The hostname of the deployed network loadbalancer"
   type = string
+}
+variable "release_sequence" {
+  description = "Pin a replicated release sequence on firstboot."
+  default = ""
```

Comment:
Hmm. If we're defaulting this in the calling module, maybe we should set a default here in the inner module?

Reply:
Yea for sure, I'll pull that out.